setup.py

```diff
@@ -36,6 +37,9 @@
 
 
 PACKAGE_DATA = {
+    "cms": [
+        "cms_conf_legacy_mapping.toml",
```

Comment:
This is a strange path to have this template, but I don't know where it would fit in. Perhaps the whole translation function should be transfered somewhere else (away from `cms/Config.py`) anyway?

Reply:
I think it's fine here, but maybe calling it `cms_conf_legacy_mapping.toml.jinja` would make more sense? What do you think?